lisp/ui/widgets/dbmeter.py

```diff
@@ -77,6 +87,35 @@ def plot(self, peaks, _, decayPeak):
 
         self.update()
 
+    def updateMarkings(self):
+        self._markings = []
+
+        height = self.height()
+        font_height = QFontMetrics(self.font()).height()
+        steps = [1, 5, 10, 25, 50, 100, 250]
+        curr_level = self.dBMax
+        curr_y = 0
+
+        while curr_y < height - font_height:
+            prev_level = curr_level
+            prev_y = curr_y + font_height
+
+            for step in steps:
+                curr_level = prev_level - step
+                curr_y = height - self.scale(curr_level) * (height - 2)
+                if curr_y > prev_y:
+                    break
+
+            self._markings.append([curr_y, curr_level])
+
+        self._markings.pop()
+
+        self.scale_width = max(
+            QFontMetrics(self.font()).boundingRect(str(-abs(self.dBMax - self.dBMin))).width(),
```

Comment:
I suppose we could assume that self.dBMin will be the "longest" value?
```suggestion
            fm.boundingRect(str(self.dBMin)).width(),
```

Reply:
Not necessarily - the third image in the PR description shows an example where that's not the case: `self.dBMin` = 0, `self.dBMax` = 40

Reply:
eheh, sorry, I've looked at it forgot :facepalm: